packages/desktop-client/src/components/FatalError.js

```diff
@@ -77,9 +77,9 @@ class FatalError extends React.Component {
         >
           <Text>{msg}</Text>
           <Text>
-            Contact{' '}
-            <a href="mailto:help@actualbudget.com">help@actualbudget.com</a> for
-            support
+            Open an{' '}
+            <a href="https://github.com/actualbudget/actual/issues">issue</a>{' '}
```

Comment:
Can you update this one too?

Reply:
Ah yes, that wasn't one of my initial changes so missed it. Updated now.